src/plugins/cache-util.js

```diff
@@ -10,8 +10,8 @@ import * as util from "../commons/util.js";
 import * as dnsutil from "../commons/dnsutil.js";
 import * as envutil from "../commons/envutil.js";
 
-const minTtlSec = 30; // 30s
-const maxTtlSec = 180; // 3m
+const minTtlSec = 300; // 30s
+const maxTtlSec = 86400; // 3m
```

Comment:
These are used in DNS packets as-is; which represent these in secs not in millis.